tools/raml2html/.gitignore

```diff
@@ -1,4 +1,3 @@
 var
 vendor
-composer.lock
```

Comment:
Still ignored by [root's .gitignore](https://github.com/ezsystems/developer-documentation/blob/2278ce29f91bab2640a08a37cb15443b810d3875/.gitignore#L7) 😞 
I should also remove it as a [best practice](https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control) but this is not the goal of this PR to change the whole practice at root level.